metadetect/detect.py

```diff
@@ -168,10 +188,16 @@ def __init__(self,
             Dict holding sep extract parameters
         meds_config: dict, optional
             Dict holding MEDS parameters
+        wcs: function, optional
+            A function with call signature `wcs(row, col)` that returns a
+            dictionary with keys {'dudrow', 'dudcol', 'dvdrow', 'dvdcol'} and
+            the corresponding values. If `None`, then the jacobian of the
+            input mbobs is used.
```

Comment:
I used a function here for the same reason above. The offset of the slice and any position offset due to coord conventions can be encapsulated in a closure.